api/models/v0.2/modelStreamPatron.js

```diff
@@ -9,7 +9,12 @@ const modelStreamPatron = {
     emails: [],
     pin: null,
     patronType: null,
-    patronCodes: null,
+    patronCodes: {
```

Comment:
So in the other pull request, https://github.com/NYPL/dgx-patron-creator-service/pull/21/files, you set each pcode to be null but in this PR you're setting it to 0. Can you explain why in one PR they all are equaling null and in another, they're 0?

Reply:
For consistency, I made the change to `null` just now.  Thanks @nnat425

Reply:
Nice.